### DISCUSSION TITLE
New Warehouse Cybercastor Runs Job File

### DISCUSSION THREAD

**Discussion by KellyMWhitehead** [Riverscapes/riverscapes-tools#783] [General] [answered]
@philipbaileynar
@MattReimer 

Please review the following for the upcoming cybercastor jobs:

- [ ] Job Name
- [ ] Job Description
- [ ] Metadata
- [ ] Warehouse tag(s)
- [ ] Org ID
- [ ] GitRef
- [ ] Anything else to add

```json
{
  "$schema": "../job.schema.json",
  "name": "RSC CA TAU Production 1701",
  "description": "RS Context, Channel Area and TauDEM production run for all HUC 10s in HUC4 1701.",
  "taskScriptId": "rs_context_channel_taudem",
  "meta": {
    "GITHUB_ISSUE": "",
    "INITIATIVE": "NRCS,CEAP"
  },
  "server": "PRODUCTION",
  "env": {
    "TAGS": "CyberCastor,NRCS,CEAP",
    "VISIBILITY": "PUBLIC",
    "ORG_ID": "b35b8f4f-016d-4c60-bbaa-11c9563fb744"
  },
  "hucs": [
    "1701010101",
    "1701010102",
    "... etc"
  ]
}
```

**Comment by MattReimer** [comment]
My contribution:

- Job Name: love it
- Job Description: Yes!
- Metadata: I'd like to see a grouping concept that we could use to tie jobs together like: `{ "JOBGROUP": "May2023Rerun" }`
- Warehouse tag(s): ??? I'm really not sure. Maybe we could just tag them `Cybercastor`  or multiples `Cybercastor,May2023`... @philipbaileynar ?
- Org ID: Riverscapes Org (Name TBD but the id will not change): `b35b8f4f-016d-4c60-bbaa-11c9563fb744`
- GIT_REF: REMOVE THIS. We're brinnging master branch up to this level today. No need for GIT_REF

**Comment by philipbaileynar** [comment, marked as answer]
Fixing some typos

```json
{
  "$schema": "../job.schema.json",
  "name": "RSC CA TAU Production 1701",
  "description": "RS Context, Channel Area and TauDEM production run for all HUC 10s in HUC4 1701.",
  "taskScriptId": "rs_context_channel_taudem",
  "meta": {
    "GITHUB_ISSUE": "",
    "INITIATIVE": "NRCS,CEAP"
  },
  "server": "PRODUCTION",
  "env": {
    "TAGS": "CyberCastor,NRCS,CEAP",
    "VISIBILITY": "PUBLIC",
    "ORG_ID": "b35b8f4f-016d-4c60-bbaa-11c9563fb744"
  },
  "hucs": [
    "1701010101",
    "1701010102",
    "... etc"
  ]
}
```

**Comment by philipbaileynar** [comment]
- I really like the name and description.
- I really like @MattReimer "job group suggestion". That helps gather together this sprint to populate the warehouse.
- The "cybercastor" tag doesn't provide much value... We are going to have 30,000 projects, 99% of which will be cybercastor. Doesn't distinguish enough to be of value.

**Reply by MattReimer** [reply]
I think we should have SOME kind of warehouse tag though. The tag is useful in the warehouse for:

1. differentiating projects of a similar type
2. Filtering the map view
3. It will be distinct from the tags we use (or don't use) on the unrunnables or QRIS or custom uploaded projects.

**Comment by KellyMWhitehead** [comment]
Final addition of metadata item Processing Group e.g. `"PROCESSING_GROUP": "1710-1712"` to encapsulate the huc4's used in the job. Just in case.
